easydock/database.py

```diff
@@ -136,6 +139,18 @@ def restore_setup_from_db(db_fname):
 
     return d, tmpfiles
 
+def get_isomers(mol):
+    opts = StereoEnumerationOptions(tryEmbedding=True,maxIsomers=32,rand=0xf00d)
```

Comment:
`max_isomers` should be added to arguments and elevated to the level of script arguments (argparse arguments) with the default value 1.

Reply:
add please spaces after the commas